soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -4239,88 +4242,127 @@ void DrawRandoEditor(bool& open) {
                         "in the base pool."
                     );
                     SohImGui::EnhancementCombobox("gRandomizeIceTraps", randoIceTraps, 5, 1);
-                    ImGui::Separator();
                     ImGui::PopItemWidth();
                     ImGui::EndTable();
                 }
+                ImGui::PopStyleVar(1);
                 ImGui::EndTabItem();
             }
 
-            if (ImGui::BeginTabItem("Detailed Logic")) {
-                if (ImGui::BeginTable("tableRandoDetailedLogic", 3,
+            if (ImGui::BeginTabItem("Location Logic")) {
```

Comment:
not sure how i feel about the name "Location Logic", i prefer just "Locations"

Reply:
I felt like this would clarify it a bit more what it's about, but considering the context (aka the other tab names surrounding it), you're probably right. I'll change this.